rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -565,83 +567,44 @@ void testGetQueryByPorSecondPageSortedByType() {
   }
 
   @Test
-  void testGetTaskWithAttachments() throws IOException {
-    final URL url =
-        new URL(restHelper.toUrl("/api/v1/tasks/" + "TKI:000000000000000000000000000000000002"));
-    HttpURLConnection con = (HttpURLConnection) url.openConnection();
-    con.setRequestMethod("GET");
-    con.setRequestProperty("Authorization", "Basic YWRtaW46YWRtaW4=");
-    assertThat(con.getResponseCode()).isEqualTo(200);
-    final ObjectMapper objectMapper = new ObjectMapper();
-
-    BufferedReader in = new BufferedReader(new InputStreamReader(con.getInputStream(), UTF_8));
-    String inputLine;
-    StringBuilder content = new StringBuilder();
-    while ((inputLine = in.readLine()) != null) {
-      content.append(inputLine);
-    }
-    in.close();
-    con.disconnect();
-    String response = content.toString();
-    JsonNode jsonNode = objectMapper.readTree(response);
-    String created = jsonNode.get("created").asText();
-    assertThat(response.contains("\"attachments\":[]")).isFalse();
-    assertThat(created.matches("\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{3}Z")).isTrue();
+  void should_NotGetEmptyAttachmentList_When_GettingTaskWithAttachment() {
+    ResponseEntity<TaskRepresentationModel> response =
+        template.exchange(
+            restHelper.toUrl(Mapping.URL_TASKS_ID,
+                "TKI:000000000000000000000000000000000002"),
+            HttpMethod.GET,
+            new HttpEntity<>(restHelper.getHeadersAdmin()),
+            ParameterizedTypeReference.forType(TaskRepresentationModel.class));
+
+    TaskRepresentationModel repModel = response.getBody();
+    assertThat(repModel).isNotNull();
+    assertThat(repModel.getCreated().matches("\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}\\.\\d{3}Z"))
```

Comment:
I think you could use assertThat(xyz).matches(pattern);

Reply:
yes, you can. See [here](http://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/AbstractCharSequenceAssert.html#matches-java.lang.CharSequence-)